scst-store/configure-access-token.hbs.md

```diff
@@ -0,0 +1,16 @@
+# Configure Access Tokens
+
+Service accounts are required to generate the access tokens.  
```

Comment:
Funny enough... this statement is not quite accurate in k8s 1.24 🤔

Reply:
I changed this parts wording, does that look better?